mlflow/store/model_registry/file_store.py

```diff
@@ -801,6 +824,61 @@ def delete_model_version_tag(self, name, version, key):
             updated_time = get_current_time_millis()
             self._update_registered_model_last_updated_time(name, updated_time)
 
+    def _get_registered_model_alias_path(self, name, alias):
+        _validate_model_name(name)
+        _validate_model_alias_name(alias)
```

Comment:
Can we validate that `alias` doesn't contain `/` (path separator) to prevent [LFI](https://brightsec.com/blog/local-file-inclusion-lfi/#:~:text=What%20is%20Local%20File%20Inclusion,XSS)%20and%20remote%20code%20execution.)?